app.rb

```diff
@@ -1,8 +1,41 @@
 require 'sinatra'
 require 'sinatra/reloader'
+require "active_record"
 require_relative 'db/connection'
 require_relative 'models/car'
 
 get '/cars' do
-  @cars = Car.all
-end
\ No newline at end of file
+  @car = Car.all
+  erb :"car/index"
```

Comment:
:+1: although the common convention is to pluralize the directory (i.e., `cars/index`)